lambdas/check-coi/src/test/java/uk/gov/di/ipv/core/checkcoi/CheckCoiHandlerTest.java

```diff
@@ -108,7 +109,8 @@ void shouldReturnPassedForSuccessfulGivenNamesOnlyCheck(
                                 List.of(M1A_ADDRESS_VC, M1A_EXPERIAN_FRAUD_VC)))
                         .thenReturn(true);
                 when(mockIpvSessionItem.getCoiSubjourneyType()).thenReturn(coiSubjourneyType);
-                when(mockClientSessionItem.getScopeClaims()).thenReturn(new String[] {"openid"});
+                when(mockClientSessionItem.getScopeClaims())
+                        .thenReturn(Arrays.asList(ScopeConstants.OPENID));
```

Comment:
(Personally I have a slight preference for `List.of()` - but it's fairly equivalent)